collectivo/extensions/shifts/server/schemas/001_schema_shifts.ts

```diff
@@ -0,0 +1,208 @@
+      schema: "schema",
+      name: "schema",
+      comment: null,
+    },
+    meta: {},
+  },
+  {
+    collection: "shifts_assignments",
+    schema: {
+      schema: "schema",
+      name: "schema",
+      comment: null,
+    },
+    meta: {},
+  },
+  {
+    collection: "shifts_logs",
+    schema: {
+      schema: "schema",
+      name: "schema",
+      comment: null,
+    },
+    meta: {},
+  },
+];
+
+schema.fields = [
+  ...directusSystemFields("shifts_shifts"),
+  {
+    collection: "shifts_shifts",
+    field: "shifts_name",
+    type: "string",
+    schema: {},
+    meta: {},
+  },
+  {
+    collection: "shifts_shifts",
+    field: "shifts_from",
+    type: "dateTime",
+    schema: { is_nullable: false },
+    meta: {},
+  },
+  {
+    collection: "shifts_shifts",
+    field: "shifts_to",
+    type: "dateTime",
+    schema: { is_nullable: true },
+    meta: {},
+  },
+  {
+    collection: "shifts_shifts",
+    field: "shifts_duration",
+    type: "integer",
+    schema: { is_nullable: false },
+    meta: { note: "In minutes" },
+  },
+  {
+    collection: "shifts_shifts",
+    field: "shifts_repeats_every",
+    type: "integer",
+    schema: { is_nullable: true },
+    meta: { note: "In days" },
+  },
+  ...directusSystemFields("shifts_slots"),
+  {
+    collection: "shifts_slots",
+    field: "shifts_name",
+    type: "string",
+    schema: {},
+    meta: {},
+  },
+  ...directusSystemFields("shifts_skills"),
+  {
+    collection: "shifts_skills",
+    field: "shifts_name",
+    type: "string",
+    schema: {},
+    meta: {},
+  },
+  ...directusSystemFields("shifts_assignments"),
+  {
+    collection: "shifts_assignments",
+    field: "shifts_from",
+    type: "date",
+    schema: {},
+    meta: {},
+  },
+  {
+    collection: "shifts_assignments",
+    field: "shifts_to",
+    type: "date",
+    schema: {},
+    meta: {},
+  },
+  {
+    collection: "directus_users",
+    field: "shifts_group",
+    type: "alias",
+    meta: {
+      order: 200,
+      interface: "group-detail",
+      special: ["alias", "no-data", "group"],
+      options: { title: "$t:person", headerIcon: "account_circle" },
+    },
+  },
+  {
+    collection: "directus_users",
+    field: "shifts_user_type",
+    type: "string",
+    schema: { is_nullable: false, default_value: ShiftUserType.TypeNotChosen },
+    meta: {
+      group: "shifts_group",
+      interface: "select-dropdown",
+      options: {
+        choices: shiftUserTypeDropdownChoices,
+      },
+    },
+  },
+  {
+    collection: "shifts_logs",
+    field: "shifts_type",
+    type: "string",
+    schema: { is_nullable: false },
+    meta: {
+      interface: "select-dropdown",
+      options: {
+        choices: shiftLogTypeDropdownChoices,
+      },
+    },
+  },
+  {
+    collection: "shifts_logs",
+    field: "shifts_datetime",
+    type: "dateTime",
+    schema: { is_nullable: false },
+    meta: {},
+  },
+];
+
+schema.createO2MRelation("shifts_slots", "shifts_shifts", "shifts_shift");
```

Comment:
Here we need to add a field on the shifts_shift side, so that you can edit the slots when you edit a shift.